src/libraries/System.Net.WebProxy/src/System/Net/WebProxy.cs

```diff
@@ -68,7 +68,14 @@ public string[] BypassList
             get { return _bypassList != null ? (string[])_bypassList.ToArray(typeof(string)) : Array.Empty<string>(); }
             set
             {
-                _bypassList = new ArrayList(value);
+                if (value == null)
+                {
+                    _bypassList = null;
```

Comment:
If we now allow null to be set, and with the constructor taking a nullable string[], seems this property should now have `[AllowNull]` on it.

Reply:
Added. Thanks!